scripts/start.sh

```diff
@@ -7,8 +7,18 @@ set -e
 ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cd "$ScriptDir"
 
+# get the port from the REACT_APP_BACKEND_URI env variable
+envContent=$(grep -v '^#' ../webapp/.env | xargs)
+backendPort=$(echo $envContent | sed -n 's/.*:\([0-9]*\).*/\1/p')
+
 # Start backend (in background)
 ./start-backend.sh &
 
+# check that the backend is running and keep checking until it is
+while ! nc -z localhost $backendPort; do
```

Comment:
Same here - would be a good idea to add a timeout in case the backend server goes dark for any reason